python/main.py

```diff
@@ -66,14 +67,21 @@ def add_item(id: int = Form(...), name: str = Form(...), category: str = Form(..
         f.write(file)
 
     # Add new items into items.db
-    conn = sqlite3.connect(dbpath)
+    conn = sqlite3.connect(sqlpath)
     cursor = conn.cursor()
-    cursor.execute("INSERT INTO items (id, name, category_id, image_name) VALUES (?, ?, ?, ?)",
-                   (id, name, category_id, filename,))
-    cursor.execute("INSERT INTO category (id, name) VALUES (?, ?)", (id, category))
+    cursor.execute("SELECT id FROM category WHERE name = ? ", (category,))
+    category_id = cursor.fetchone()
+    if category_id is None:
+        cursor.execute("INSERT INTO category (name) VALUES (?)", (category,))
+        conn.commit()
+        category_id = cursor.fechone()[0]
```

Comment:
fechone is typo and correct one is fetchone. However, fetchone doen't work here because previous query is insert, not select.

Could you check and use this instead of these lines?
https://docs.python.org/3/library/sqlite3.html#sqlite3.Cursor.lastrowid

Also, when you write codes, it is better to test it and check how it works. 👍